### DISCUSSION TITLE
Header missing

### DISCUSSION THREAD

**Discussion by blockchainDevCDST** [jsvine/pdfplumber#978] [Q&A]
<img width="946" alt="image" src="https://github.com/jsvine/pdfplumber/assets/111956715/85a608f9-b704-472d-b86e-2f4422a35096">

I have a table like above but when I do extract_table it is not returning header in table. Any suggestions how to get it?

**Comment by samkit-jain** [comment]
Hi @blockchainDevCDST Appreciate your interest in the library. Could you please provide the table extraction settings that you are using? My guess would be that you are using `lines` for the `horizontal_strategy`. Recommend you to try using `text`. You can also have a post-processing step that captures the header above the table automatically if not captured.

To assist you further, request you to please share the PDF as well.

**Comment by mmaysami** [comment]
Hey @samkit-jain, also having similar table extraction issue where the bold headers in first couple of rows are ignored. I am working on a table in format on file below. Do you have any suggestion on what would best approach to extend the table to include the headers here?

[myfile_1.pdf](https://github.com/jsvine/pdfplumber/files/13572512/myfile_1.pdf)

![visual](https://github.com/jsvine/pdfplumber/assets/23468747/b6b00862-7acb-420a-9bd5-80d1d4e22fb3)

**Reply by samkit-jain** [reply]
Hi @mmaysami One option could be to define the explicit vertical lines and use the "text" strategy for horizontal lines. However, this approach would mean that you would have to do some post-processing to clean up the results.
